README.md

```diff
@@ -21,11 +21,20 @@ sudo sensors-detect
 sudo service kmod start
 ```
 
-### Raspberry Pi Raspbian
+### On Raspberry Pi Raspbian
 
 This plugin uses `vcgencmd measure_temp` command on RPi Raspbian to get CPU temperature.
 Should be compatible with other RPi distros with `vcgencmd` available.
 
+### On macOS
+
+This plugin uses [lavoiesl/osx-cpu-temp](https://github.com/lavoiesl/osx-cpu-temp).
+
+```console
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/narugit/temp-cpu/master/install_requirements.sh)"
```

Comment:
I tried the link but it does not work for me. I think better than adding the script `install_requirements.sh` file, just include a few lines about how to install osx-cpu-temp utility. For example:
```
git clone https://github.com/lavoiesl/osx-cpu-temp
cd osx-cpu-temp
make && sudo make install)
```

Reply:
Thank you for checking and I am sorry that the link does not worked.

> I think better than adding the script install_requirements.sh file, just include a few lines about how to install osx-cpu-temp utility. 

I agree with you! I am going to update.